db.py

```diff
@@ -13,7 +15,11 @@
 
 
 def get_db_connection():
-    db_url = os.getenv('DATABASE_URL')
+    if os.getenv('FLASK_ENV') == 'TEST':
+        db_url = os.getenv('TEST_DATABASE_URL')
```

Comment:
Why the env variable is different?

Reply:
At the beginning when i tried to write tests for the APIs i realized that i need to separate the real DB from the testing DB so i though an easy solution will be to hold a variable in my .env file that will tell my server which db to use.

Now I added the test_app.py in which i use to mock the desired tests so this code is now redundant.

Reply:
Still, test_app.py can use the get_db_connection() you just need to set different env variable in your test env :)